cli/lib/kontena_cli.rb

```diff
@@ -158,11 +159,14 @@ def self.logger
       require 'kontena/cli/log_formatters/strip_color'
       logger.formatter = Kontena::Cli::LogFormatter::StripColor.new
     end
-    logger.level = ENV["DEBUG"] ? Logger::DEBUG : Logger::INFO
+    logger.level = debug? ? Logger::DEBUG : Logger::INFO
     logger.progname = 'CLI'
     @logger = logger
   end
 
+  def self.debug?
+    !ENV['DEBUG'].nil? && ENV['DEBUG'].to_s != 'false'
```

Comment:
`DEBUG=` -> `ENV['DEBUG'].empty?` should also be false.

Reply:
true